### PR TITLE
Add `Client/Interface/Chat console/Highlight`

### PR DESCRIPTION
**Pull request by TicClick**
felt like this one was missing when I was translating the help centre pages

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Client/Interface/Chat_console/Highlight/en.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - mention
+  - hilight
+---
+
+# Highlight
+
+**Highlight** (sometimes called a **mention**) is a way to draw someone's attention to the chat if they are online. By default, highlight is triggered when a player's username is written out in the chat. The list of additional highlight words may be customized in osu!'s [in-game chat options](/wiki/Client/Options#in-game-chat).
```
Comment (Walavouchey) [inline wiki/Client/Interface/Chat_console/Highlight/en.md:9]:
```suggestion
A **highlight** (commonly abbreviated as **HL**, sometimes called a **mention** or **ping**) is a way to draw someone's attention to the chat if they are online. By default, a highlight is triggered when a player's username is written out in the chat. The list of what words get highlighted may be customised in osu!'s [in-game chat options](/wiki/Client/Options#in-game-chat).
```

File: wiki/Client/Interface/Chat_console/Highlight/en.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - mention
+  - hilight
```
Comment (Walavouchey) [inline wiki/Client/Interface/Chat_console/Highlight/en.md:4]:
```suggestion
  - mention
  - hilight
  - HL
  - ping
```

File: wiki/Client/Interface/Chat_console/Highlight/en.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+It is possible to ignore highlights from specific users by adding their usernames to the [ignore list](/wiki/Client/Options/Ignore_list) with `@h` appended to them.
+
+## Appearance
+
+::: Infobox
+![](img/hl-notification.png "An example of highlight notification")
+:::
+
+When a chat message triggers highlight, the sender's username on that line becomes green, and the message is added to a `#highlight` chat tab. Additionally, if the osu!'s window is inactive, its icon will flash on the taskbar.
```
Comment (Walavouchey) [inline wiki/Client/Interface/Chat_console/Highlight/en.md:19]:
```suggestion
When a chat message triggers highlight, the sender's username on that line becomes green, and the message is added to a `#highlight` chat tab. Additionally, if the osu! window is inactive, its icon will flash on the taskbar.
```

File: wiki/Skinning/Sounds/fr.md [hunk 4]
```diff
@@ -447,7 +447,7 @@ Notes :
 Notes :
 
 - Ce son est joué lorsque le match commence.
-- Il s'agit également du son des notifications dans le tchat.
+- Il s'agit également du son des [notifications](/wiki/Client/Interface/Chat_console/Highlight) dans le tchat.
```
Comment (Walavouchey) [inline wiki/Skinning/Sounds/fr.md:450]:
```suggestion
- Il s'agit également du son des [notifications dans le tchat](/wiki/Client/Interface/Chat_console/Highlight).
```

File: wiki/Client/Interface/Chat_console/Highlight/en.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+# Highlight
+
+**Highlight** (sometimes called a **mention**) is a way to draw someone's attention to the chat if they are online. By default, highlight is triggered when a player's username is written out in the chat. The list of additional highlight words may be customized in osu!'s [in-game chat options](/wiki/Client/Options#in-game-chat).
+
+It is possible to ignore highlights from specific users by adding their usernames to the [ignore list](/wiki/Client/Options/Ignore_list) with `@h` appended to them.
+
+## Appearance
+
+::: Infobox
+![](img/hl-notification.png "An example of highlight notification")
```
Comment (Walavouchey) [inline wiki/Client/Interface/Chat_console/Highlight/en.md:16]:
```suggestion
![](img/hl-notification.png "An example of a highlight notification")
```

File: wiki/Client/Interface/Chat_console/Highlight/en.md (hunk 2 above)
Comment (TicClick) [reply wiki/Client/Interface/Chat_console/Highlight/en.md:4]: huh, really

File: wiki/Client/Interface/Chat_console/Highlight/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Client/Interface/Chat_console/Highlight/en.md:9]: going to accept it for the sake of simplicity, but words aren't getting highlighted, the whole line does

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**